<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="KRISH107" failures="1" tests="2" name="Local Tests" time="367.055" errors="0" timestamp="2024-09-25T16:58:03 IST">
  <testcase classname="sanity.bateel.SanityBateelLiveCheckout" name="testBateelLiveCheckout" time="186.346"/>
  <testcase classname="sanity.bateel.SanityBateelLiveCheckoutWithInvalidCardDetails" name="testBateelLiveCheckoutWithInvalidCard" time="173.134">
    <failure type="java.lang.AssertionError" message="expected [The customer was not redirected as expected.] but found [Customer should be redirected to the visa page]">
      <![CDATA[java.lang.AssertionError: expected [The customer was not redirected as expected.] but found [Customer should be redirected to the visa page]
at page.bateel.BateelLoginCheckoutPage.handleErrorMessage(BateelLoginCheckoutPage.java:132)
at sanity.bateel.SanityBateelLiveCheckoutWithInvalidCardDetails.testBateelLiveCheckoutWithInvalidCard(SanityBateelLiveCheckoutWithInvalidCardDetails.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testBateelLiveCheckoutWithInvalidCard -->
</testsuite> <!-- Local Tests -->
